test/core_language/test/unicode_test.gleam

```diff
@@ -1,5 +1,16 @@
 import should
 
+pub fn a_bunch_of_unicode_test() {
+  let unicode = "👨‍👩‍👧‍👧"
```

Comment:
Seeing as this test doesn't require any code to be run we could test this in the normal compiler tests.

Perhaps `parser::strip_extra_test` as that tests the code that is crashing here.